gandalf.js

```diff
@@ -1,10 +1,11 @@
 /**
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?